buttons/button_base.h

```diff
@@ -50,6 +59,13 @@ class ButtonBase : public Looper,
       }
       while (DebouncedRead()) YIELD();
       prop.Event(button_, EVENT_RELEASED);
+      if (doubleclicked_) {
+        if (!doubleclickedandheld_) {
+          prop.Event(button_, EVENT_DOUBLE_CLICK);
+        }
+        doubleclicked_ = false;
+      }
+      doubleclickedandheld_ = false;
```

Comment:
If the button is released and a HELD event wasn't triggered, the DOUBLE_CLICK event fires.  The two bools used to control the doubleclick behavior are reset.